package.json

```diff
@@ -5,18 +5,31 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Update README with running storybook.

Reply:
Updated in a39ed923d1bc5f213e17618e597c2cc4575c75df